examples/multigpu_advtrain/cifar_input.py

```diff
@@ -0,0 +1,154 @@
+# From RevNets code
```

Comment:
Should we put a specific URL?
Is the whole file from the other repo?
What license does the other repo use? We probably need to copy their copyright statement

Reply:
License and url added in 7c0b864